cfme/infrastructure/provider.py

```diff
@@ -231,6 +237,9 @@ def validate(self, db=True):
         a set of statistics to be matched against the UI. The details page is then refreshed
         continuously until the matching of all items is complete. A error will be raised
         if the match is not complete within a certain defined time period.
+
+        Raises:
```

Comment:
I think this doc should be moved into the function that does "actually" raise it, though from an API PoV I see why it's here.